File: PhP/tempCodeRunnerFile.php
<?php
// Solicitar al usuario el primer valor
echo "Ingrese el primer valor: ";
$A = intval(trim(fgets(STDIN))); // Leer entrada y convertir a entero

// Solicitar al usuario el segundo valor
echo "Ingrese el segundo valor: ";
$C = intval(trim(fgets(STDIN))); // Leer entrada y convertir a entero

// Calcular la potencia A^C
$potencia = pow($A, $C); // Función pow(base, exponente) en PHP

// Mostrar el resultado
echo "La potencia de $A sobre $C es: $potencia\n";

File: C mas/tempCodeRunnerFile.cpp
#include <iostream> // Biblioteca para entrada/salida
#include <cmath>    // Biblioteca para operaciones matemáticas como pow()

using namespace std;

int main()
{
    int A, C; // Declaramos variables para almacenar los valores ingresados

    // Solicitar el primer valor
    cout << "Ingrese el primer valor: ";
    cin >> A; // Leer el número entero

    // Solicitar el segundo valor
    cout << "Ingrese el segundo valor: ";
    cin >> C; // Leer el número entero

    // Calcular la potencia A^C usando pow()
    int potencia = pow(A, C);

    // Mostrar el resultado
    cout << "La potencia de " << A << " sobre " << C << " es: " << potencia << endl;

    return 0; // Indicar que el programa terminó correctamente
}
